tests/fixtures/html/00646-vor-section-references.html

```diff
@@ -0,0 +1 @@
+<div class="elife-reflinks-links"><article class="elife-reflinks-reflink" id="bib1" data-original="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="1">1</span></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="http://www.nhs.uk/conditions/hip-fracture/Pages/introduction.aspx" target="_blank"><span class="nlm-article-title">Hip fracture</span></a></cite><div class="elife-reflink-authors"><span class="nlm-collab">NHS Choices</span></div><div class="elife-reflink-details"><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-uri"><a href="http://www.nhs.uk/conditions/hip-fracture/Pages/introduction.aspx" target="_blank">http://www.nhs.uk/conditions/hip-fracture/Pages/introduction.aspx</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="bib2" data-original="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="2">2</span></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="http://dx.doi.org/10.1371/journal.pone.0038466" target="_blank"><span class="nlm-article-title">Cortical thickness mapping to identify focal osteoporosis in patients with hip fracture</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:KES Poole&quot;" target="_blank">KES Poole</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:GM Treece&quot;" target="_blank">GM Treece</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:PM Mayhew&quot;" target="_blank">PM Mayhew</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:J Vaculik&quot;" target="_blank">J Vaculik</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:P Dungi&quot;" target="_blank">P Dungi</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Horák&quot;" target="_blank">M Horák</a></span> et al. </div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source">PLOS One</span></span>, <span class="elife-reflink-details-volume">7</span>, <span class="elife-reflink-details-pages">e38466</span>, <span class="elife-reflink-details-year">2012</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0038466" target="_blank">http://dx.doi.org/10.1371/journal.pone.0038466</a></span></div></div></div></article></div>
```

Comment:
This file is named wrong but it already exists: https://github.com/elifesciences/elife-ingest-xsl/blob/dev/tests/fixtures/html/00646-v1-vor-section-references.html